static/app/components/codeSnippet.tsx

```diff
@@ -1,24 +1,34 @@
-import {Fragment, useEffect, useRef, useState} from 'react';
-import styled from '@emotion/styled';
+// Prism components need to be imported after Prism
+// eslint-disable-next-line simple-import-sort/imports
 import Prism from 'prismjs';
+import 'prismjs/components/prism-bash.min';
```

Comment:
Should this only be loaded where bash syntax highlighting is used, instead of always in this component?

Reply:
Sorry that's an outdated diff, the changes to `codeSnippet` were merged to master in a different PR.

Originally I thought having bash syntax always available would be pretty helpful, since we have a lot of bash snippets in the app (specifically for installation code, and some markdown user comments may contain bash). On further thought, though, given that Prism's bash bundle is non-trivial in size (about 6kB minified), I think you have a good point about localizing the bundle imports. I'll submit a new PR to do that!